lib/carin_for_blue_button_test_kit/client/v2.0.0/c4bb_client_test_suite.rb

```diff
@@ -113,5 +120,18 @@ def self.test_resumes?(test)
       test from: :eob_professional_claims_data_request_test
       test from: :client_claims_data_attestation
     end
+
+    group do
```

Comment:
We have not discussed this, but I do think this would make a lot of sense to have only a single wait test. As for how to split the tests up, it still might be good to have the MS and search tests split, so that a user can run only one type of test if they want. This would allow the reference client to run only against the MS tests, which it can successfully run and pass, without needing to run against the search tests. It also makes the attestation test at the end of the MS group of tests make more sense. But this may not be necessarily the best reason to separate the tests like this, so if you think splitting by resource makes more sense, I can definitely do that! At the very least, I do think it makes sense to have a single wait

Reply:
I think what you're suggesting is the following structure:

- wait test
- searches group
   - profile A
       - at least one search test
       - all required searches test
   - profile B ...
- must support group
   - profile A must support test
   - profile B must support test ...

Your reasoning is that users could run the wait test and then the must support group if they didn't want to demonstrate all searches, e.g., because they couldn't.

I guess my inclination is to group them with the other search tests. The profile groups don't have to be run as group, so testers could skip specific tests, but making it easy to run just must support tests in one click doesn't seem like a big win from a tester experience perspective.